test/test-pages/wapo-1/expected.html

```diff
@@ -1,5 +1,37 @@
 <div id="readability-page-1" class="page">
     <article>
+        <div data-category="Foreign" data-commercial-node="world" data-first-published="1426686630" data-permalink="http://www.washingtonpost.com/world/at-least-8-dead-in-tunisia-attack-on-leading-museum/2015/03/18/ef22654e-cd72-11e4-8c54-ffb5ba6f2f69_gallery.html" data-preroll-zone="" data-published="1426713600" data-section="world" data-show-interstitials="true" data-show-preroll="true" data-slug="at-least-8-dead-in-tunisia-attack-on-leading-museum" data-subsection="" data-title="At least 17 dead in Tunisia after attack in capital city" data-uuid="ef22654e-cd72-11e4-8c54-ffb5ba6f2f69" data-pb-content-uri="/pb/galleryEmbed/ef22654e-cd72-11e4-8c54-ffb5ba6f2f69" data-pb-name="gallery-gallery" id="gallery-embed_1417452270618_709">
+            <div>
+                <p>Full Screen</p>
+                <p>Autoplay</p>
+                <p>Close</p>
+            </div>
+            <div>
+                <p><span>At least 17 dead in Tunisia after attack in capital city</span>
+                </p>
+                <div>
+                    <p><img src="https://img.washingtonpost.com/rf/image_606w/2010-2019/WashingtonPost/2015/03/18/Foreign/Images/Nic6429732.jpg" />
+                    </p>
+                    <p><i></i>View Photos</p>
+                </div>
+                <p><span>Security forces in Tunisia stormed the country’s most prominent museum, freeing hostages and ending a siege by gunmen.</span>
+                </p>
+            </div>
+            <div>
+                <p>Caption</p>
+                <div>
+                    <p>Security forces in Tunisia stormed the country’s most prominent museum, freeing hostages and ending a siege by gunmen.</p>
+                    <p><span>March 18, 2015</span> <span>Tunisian security forces secure the area after gunmen attacked the famed Bardo National Museum in Tunis, the capital.</span>
+                        <span>Fethi Belaid/AFP/Getty Images</span>
+                    </p>
+                    <p><a href="#" target="_blank">Buy Photo</a>
+                    </p>
+                </div>
+            </div>
+            <div>
+                <p>Wait 1 second to continue.</p>
```

Comment:
Including text like "wait 1 second to continue" and "full screen", "autoplay", "view photos" etc. (but without working links) isn't great. The carousel stuff here and in some of the other pages really looks like it would need more work to make it appear in a reasonable way in reader mode / readability.